aptos-move/move-examples/shared_account/sources/shared_account.move

```diff
@@ -0,0 +1,139 @@
+// This module demonstrates a basic shared account that could be used for NFT royalties
+// Users can (1) create a shared account (2) disperse the coins to multiple creators
+module shared_account::SharedAccount {
+    use std::errors;
+    use std::signer;
+    use std::vector;
+    use AptosFramework::Account;
+    use AptosFramework::Coin;
+
+    // struct Share records the address of the share_holder and the numerator and the denominator of a fraction
+    struct Share has store {
+        share_holder: address,
+        numerator: u64,
```

Comment:
nit: Name this num_shares to make it clearer?

Reply:
good point! updating